src/lib/mina.ts

```diff
@@ -459,11 +471,11 @@ function LocalBlockchain({
     setTimestamp(ms: UInt64) {
       networkState.timestamp = ms;
     },
-    setGlobalSlot(slot: UInt32) {
-      networkState.globalSlotSinceGenesis = slot;
-    },
-    setGlobalSlotSinceHardfork(slot: UInt32) {
-      networkState.globalSlotSinceHardFork = slot;
+    incrementGlobalSlot(increment: UInt32 | number) {
```

Comment:
improvement to the API of setting the global slot: `globalSlotSinceGenesis` and `globalSlotSinceHardfork` should always be updated together, not independent of each other. Also, the only update that makes sense is increasing them.

Reply:
Does it matter to add `UInt64` to this type as well?

Reply:
I don't think so, slots are always just UInt32 in Mina